lib/transaction.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'tracker'
+require 'date'
+
+class Transaction
```

Comment:
Transaction has two responsibilities at the moment.  What are they? Could they be separated?